### PR TITLE
cab-meeting-2022-08

### PR DESCRIPTION
**Pull request by diegoonolasco**

### CODE REVIEW

File: content/minutes/cab/2022-08-17-minutes.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* D. Mobley announced that the Rosemary + Tresadern manuscript is in the final stages of revision and asked industrial partners who had not yet made their comments to do so. The deadline is September 17th.
+
+#### Boehringer Ingelheim and Merck are discontinuing their support to OpenFF
+
+* D. Nolasco announced that Boehringer Ingelheim and Merck are discontinuing their support to OpenFF, but will continue supporting OpenFE.
+
+#### New version of Sage (openff-2.1.0)
+
+* D. Nolasco held a consultation with board members to see if there is any interest in the release a new version of Sage (openff-2.1.0). The present members pointed out that a new version would not add value at the moment, but suggested that we seek to understand what are the gaps in science about force fields that we could attack in the interest of continuing to add value to the molecular simularion community.
```
Comment (j-wags) [inline content/minutes/cab/2022-08-17-minutes.md]:
```suggestion
* D. Nolasco held a consultation with board members to see if there is any interest in the release a new version of Sage (openff-2.1.0). The present members pointed out that a new version would not add value at the moment, but suggested that we seek to understand what are the gaps in science about force fields that we could attack in the interest of continuing to add value to the molecular simulation community.
```

File: content/minutes/cab/2022-08-17-minutes.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+#### Follow up workshops
+
+* J. Wagner announced that the follow up workshops will be scheduled for September and October. We will probably held three workshops: BespokeFit, Interchange and Interactive force field hacking - protein parameterization with PTMs.
```
Comment (j-wags) [inline content/minutes/cab/2022-08-17-minutes.md]:
```suggestion
* J. Wagner announced that the follow up workshops will be scheduled for September and October. We will probably held three workshops: BespokeFit, Interchange, and Protein parameterization with post translational modifications.
```

File: content/minutes/cab/2022-08-17-minutes.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+#### New version of the annual report
+
+* D. Mobley announced the availability of a new version of the annual report. The document addresses some shortcomings pointed out by Katharina Meier and Thomas Fox at the last governing board meeting.
```
Comment (j-wags) [inline content/minutes/cab/2022-08-17-minutes.md]: Are we posting a public copy of the annual report anywhere? Could be good to link it in this line if so.

**Review by j-wags — APPROVED** [review]
Looks good. I proposed two text changes which you should accept. I also suggested to link to the annual report if possible.

File: content/minutes/cab/2022-08-17-minutes.md (hunk 3 above)
Comment (diegoonolasco) [reply content/minutes/cab/2022-08-17-minutes.md]: I'll link the Zenodo link to the old version, and update it as soon as we have the new version available.

**Pull request merged**